src/utils/compareDate.js

```diff
@@ -0,0 +1,9 @@
+import { MILLI_SECONDS_IN_A_DAY } from "src/common/constants";
+
+export function getDaysToCompleteTask(startDate, endDate) {
+  const millisecondsInADay = MILLI_SECONDS_IN_A_DAY;
+  const diffInMilliseconds = Math.abs(endDate - startDate);
```

Comment:
Why are we using Math.abs() if it's already evident that the end date will be greater than the start date?